openshift-maven-plugin/plugin/src/main/java/org/eclipse/jkube/maven/plugin/mojo/build/AbstractDockerMojo.java

```diff
@@ -428,18 +429,16 @@
     @Override
     public void contextualize(Context context) throws ContextException {
         plexusContainer = ((PlexusContainer) context.get(PlexusConstants.PLEXUS_KEY));
-        authConfigFactory = new AuthConfigFactory();
     }
 
     @Override
     public void execute() throws MojoExecutionException, MojoFailureException {
         if (!skip) {
             boolean ansiRestore = Ansi.isEnabled();
             log = new AnsiLogger(getLog(), useColorForLogging(), verbose, !settings.getInteractiveMode(), getLogPrefix());
+            authConfigFactory = new AuthConfigFactory(log);
 
             try {
-                authConfigFactory.setLog(log);
-                imageConfigResolver.setLog(log);
```

Comment:
This one was lost, we should keep it. (probably also move it out of the try-catch block for consistence)